openmrs/apps/ipd/app.json

```diff
@@ -25,7 +25,30 @@
                 "allowFutureDates": true
             }
         },
+        "patientForwardUrl": "../clinical/index.html#/default/patient/{{patientUuid}}/dashboard?currentTab=DASHBOARD_TAB_GENERAL_KEY",
         "hideStartNewVisitPopUp": false,
+        "colorForTags": [
```

Comment:
Is this required for this PR?

Reply:
@bsneha90 Its not part of the PR. Will raise another PR with the required changes